src/apiProxy/apiConfigCreators.js

```diff
@@ -40,12 +40,11 @@ export function group(params) {
  * 	`apiMethod: 'unfollow' - POST unfollow
  */
 export function groupCommunication(params) {
-	const { urlname, conversationId, apiMethod } = params;
+	const { chapterUrlName, apiMethod } = params;
 
 	const endpoint = [
-		urlname,
+		chapterUrlName, // group communication uses an inconsistent param for chapter urlname
```

Comment:
let's change the mup-web route definition to send a `urlname` param instead of `chapterUrlName` - I really want to keep the platform as internally consistent as possible.